<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>1066</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time and Date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="dayBeginsAtLabel">
         <property name="text">
          <string>Day begins at:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KPIM::KTimeEdit" name="kcfg_DayBegins"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="useHolidayRegionLabel">
         <property name="text">
          <string>Use holiday region:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="kcfg_HolidayRegion"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="workingPeriodLabel">
         <property name="text">
          <string>Working Period:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="workingPeriodWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="workingPeriodMonday">
            <property name="text">
             <string>Monday</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="workingPeriodTuesday">
            <property name="text">
             <string>Tuesday</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="workingPeriodWednesday">
            <property name="text">
             <string>Wednesday</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="workingPeriodThursday">
            <property name="text">
             <string>Thursday</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="workingPeriodFriday">
            <property name="text">
             <string>Friday</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="workingPeriodSaturday">
            <property name="text">
             <string>Saturday</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="workingPeriodSunday">
            <property name="text">
             <string>Sunday</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KPIM::KTimeEdit" name="kcfg_DailyStartingHour"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="dailyStartingHourLabel">
         <property name="text">
          <string>Daily starting hour:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="dailyEndingHourLabel">
         <property name="text">
          <string>Daily ending hour:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KPIM::KTimeEdit" name="kcfg_DailyEndingHour"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="kcfg_ExcludeHolidays">
       <property name="text">
        <string>Exclude holidays</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>New Events</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="defaultAppointmentTimeLabel">
         <property name="text">
          <string>Default appointment time:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KPIM::KTimeEdit" name="kcfg_DefaultAppointmentTime"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="defaultDurationOfNewAppointmentLabel">
         <property name="text">
          <string>Default duration:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="kcfg_DefaultAppointmentDuration">
         <property name="displayFormat">
          <string>hh:mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="defaultReminderTimeLabel">
         <property name="text">
          <string>Default reminder time:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="defaultReminderTimeWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="kcfg_ReminderDefaultTime"/>
          </item>
          <item>
           <widget class="QComboBox" name="kcfg_ReminderDefaultUnit">
            <item>
             <property name="text">
              <string>minute(s)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hour(s)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>day(s)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="kcfg_RemindersForNewEvents">
       <property name="text">
        <string>Enable reminders for new appointments</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Views</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="kcfg_TodosUseCategoryColors">
       <property name="text">
        <string>To-dos use category colors</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="hourSizeLabel">
         <property name="text">
          <string>Hour size:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="kcfg_HourSize">
         <property name="suffix">
          <string> pixels</string>
         </property>
         <property name="minimum">
          <number>4</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="kcfg_ShowIconsInAgendaView">
       <property name="text">
        <string>Show icons in agenda view</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="kcfg_ShowTodosInAgendaView">
       <property name="text">
        <string>Show to-dos in agenda view</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="kcfg_ShowCurrentTimeLine">
       <property name="text">
        <string>Show current time line</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QFormLayout" name="formLayout_4">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <widget class="KComboBox" name="kcfg_AgendaViewColorUsage"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="agendaViewColorOptionsLabel">
         <property name="text">
          <string>Agenda view color usage:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="kcfg_ColorBusyDaysInAgendaView">
       <property name="text">
        <string>Color busy days in agenda view</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="kcfg_ShowTodosInMonthView">
       <property name="text">
        <string>Show to-dos in month view</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="monthViewColorUsageLabel">
         <property name="text">
          <string>Month view color usage:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="kcfg_MonthViewColorUsage"/>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="kcfg_ColorBusyDaysInMonthView">
       <property name="text">
        <string>Color busy days in month view</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Colors</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Holiday color:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KColorButton" name="kcfg_HolidayColor"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Agenda view background color:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KColorButton" name="kcfg_AgendaViewBackgroundColor"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Busy days background color:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KColorButton" name="kcfg_BusyDaysBackgroundColor"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Agenda view current time line color:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KColorButton" name="kcfg_AgendaViewTimeLineColor"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Working hours color:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KColorButton" name="kcfg_WorkingHoursColor"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>To-do due today color:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KColorButton" name="kcfg_TodoDueColor"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>To-do overdue color:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="KColorButton" name="kcfg_TodoOverdueColor"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Group Scheduling</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="kcfg_UseGroupwareCommunication">
       <property name="text">
        <string>Use Groupware communication</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPIM::KTimeEdit</class>
   <extends>QComboBox</extends>
   <header>libkdepim/ktimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
